### PR TITLE
Add fast unstake pallet

### LINKED ISSUE #17: Add fast unstake pallet
- [pallet_fast_unstake](https://paritytech.github.io/polkadot-sdk/master/pallet_fast_unstake/index.html)
- [info](https://support.polkadot.network/support/solutions/articles/65000169433)

### PR DESCRIPTION
**Pull request by alwayscolorblind**
Resolves #17

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -1036,6 +1036,27 @@ impl pallet_staking::Config for Runtime {
...
+    pub const BatchSize: u32 = 64;
+    pub const MaxErasToCheckPerBlock: u32 = 1;
+}
+
+// fast unstake
+impl pallet_fast_unstake::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type Deposit = Deposit;
+    type ControlOrigin = EitherOfDiverse<
```
Comment (ales-tsurko) [inline runtime/src/lib.rs:1050]: Please, check this. Maybe it's better to delegate it to technical committee than council.

File: Cargo.lock [hunk 2]
```diff
@@ -2740,7 +2741,7 @@ checksum = "00b0228411908ca8685dba7fc2cdd70ec9990a6e753e89b6ac91a84c40fbaf4b"
 [[package]]
 name = "fork-tree"
 version = "12.0.0"
-source = "git+https://github.com/paritytech/polkadot-sdk?branch=release-polkadot-v1.8.0#ec7817e5adc2f3a91dd94b0465dd61b4c1b07ab7"
+source = "git+https://github.com/paritytech/polkadot-sdk?branch=release-polkadot-v1.8.0#a51b5dd3dd7d00b0c0ae5bdd28650eb13f0ae70c"
```
Comment (hrls) [inline Cargo.lock:2744]: Release tag was moved by parity

**Comment by hrls** [comment]
Rebase onto https://github.com/Atleta-network/atleta/pull/115

**Review by ales-tsurko — DISMISSED** [review]

**Review by DmitriVT — APPROVED** [review]

**Comment by hrls** [comment]
> Rebase onto #115

Already in `devnet`

**Pull request merged**